src/main/java/Sysc4806Group/demo/repositories/RoleRepository.java

```diff
@@ -0,0 +1,13 @@
+package Sysc4806Group.demo.repositories;
+
+import java.util.Optional;
+
+import Sysc4806Group.demo.entities.Role;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+
+@Repository
+public interface RoleRepository extends JpaRepository<Role, Long> {
```

Comment:
Is a whole repo necessary for roles? Can't it just be a boolean?

Reply:
It's just 2 records for ROLE_USER and ROLE_ADMIN. What happens is that the tables are joined to create user_roles which reference PK of user and PK and role. Boolean isn't a scalable approach.

Reply:
Sounds good